test-data/materials-broken.not-json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
optional: For this test, it might be more valuable to have a valid JSON file that is not the expected schema, which seems like a more likely error that someone would do.

Reply:
That is actually a _different_ failure mode. I'll add some code that checks for correctness as well.